ajax/libs/3Dmol/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "3Dmol",
+  "filename": "3Dmol-min.js",
+  "version": "1.0.6",
+  "description": "Object oriented Javascript molecular visualization library",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/3dmol/3Dmol.js.git"
+  },
+  "keywords": [
+    "3Dmol.js",
+    "webGL"
+  ],
+  "author": "David Koes and contributors",
+  "homepage": "http://3dmol.csb.pitt.edu",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/3dmol/3Dmol.js.git",
+    "basePath": "release",
+    "files": [
+      "**/*"
+    ]
+  },
+  "licenses": [
+    "MIT",
+    "LGPL-3.0",
+    "BSD-3-Clause"
```

Comment:
hmmm ... so there are some codes come from three.js and GLmol, so 3Dmol will be released under multiple licenses, am I correct?